cpu/cc26x0/periph/rfc.c

```diff
@@ -58,8 +58,12 @@ void rfc_irq_disable(void)
     NVIC_DisableIRQ(RF_HW_IRQN);
 }
 
-uint32_t rfc_send_cmd(void *ropCmd)
+uint32_t rfc_send_cmd(_radio_op_command_t *ropCmd)
```

Comment:
I used `void` because I was planning to build a single function that would handle both radio operation and immediate commands. In hindsight, using `ropCmd` with `void*` is inconsistent; moreover handling immediate commands isn't necessary yet. All things considered, ` _radio_op_command_t` makes things clearer and is sufficient for now.